frontend/src/components/Profile/ProfilePage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Switch from '@mui/material/Switch';
+import { Typography } from '@mui/material';
```

Comment:
reduce imports, make some's like: import { Typography, IconButton,  FormGroup ... } from '@mui/material';

Reply:
maybe you should install and use auto-import extension in vscode

Reply:
Done